sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statsEstimation/FilterEstimation.scala

```diff
@@ -311,6 +311,17 @@ case class FilterEstimation(plan: Filter) extends Logging {
       logDebug("[CBO] No statistics for " + attr)
       return None
     }
+
+    attr.dataType match {
+      case StringType | BinaryType =>
+        None
```

Comment:
This doesn't seem to do anything; is it meant to be `return None`? If it's a no-op, then it seems like this can just be omitted

Reply:
thank for your review, done